### ISSUE TITLE
No code execution - MSVCP140.dll

### ISSUE THREAD

**Issue by Phil-anthropy** [taviso/ctftool#10]
As standard untrusted user running ctftool on Windows 10 VM Version 1809 (OS Build 17763.379) errors -

"The code execution cannot proceed because MSVCP140.dll was not found. Reinstalling the program may fix this problems."

"The code execution cannot proceed because vcruntime140.dll was not found. Reinstalling the program may fix this problem."

Need to include dll dependencies in makefile?

Thanks.

**Comment by KillerRyuk** [comment]
Did you run the .exe or compile your own?

**Comment by evanmiller2112** [comment]
I am seeing the same error on an essentially out-of-the-box Win 10 1903 with the .exe from the latest release when I run it via command prompt.

If I run it int PowerShell I get no errors but it returns to the next prompt. 

It looks like that's in Visual C++ 2015? Is the redistributable a dependency?

**Comment by KillerRyuk** [comment]
Could be.  I know it is installed on the machine I tested on.  (10 1903)

**Comment by evanmiller2112** [comment]
I fixed it by installing the **x86** version of the Visual C++ 2015 Runtime (I installed the x64 version first and that did not resolve it, which makes sense I suppose as those are different DLLs if I'm not mistaken)

https://www.microsoft.com/en-us/download/details.aspx?id=52685

**Comment by taviso** [comment]
Hmm, interesting thanks - I didn't know that was a dependency, I'll try to reproduce.

**Comment by cloudsbyzeus** [comment]
Yes they are a dependency, had to install them as well, but for a standard user to execute, the binaries need to be included. https://stackoverflow.com/questions/16779736/include-visual-c-runtime-with-application. It also depends on OS version so multiple files need to be included.

**Comment by brink668** [comment]
I can confirm we just ran into the same issue as well on vanilla 1903 x64 build and when installing x86/x64 c++ runtime it worked.
https://www.microsoft.com/en-us/download/details.aspx?id=52685

**Comment by mirsella** [comment]
hello, got the same problem when i build it by myself, but i can't install this runtime https://www.microsoft.com/en-us/download/details.aspx?id=52685 who make everythings work fine. but if i use the pre build .exe, not error message because windows defender remove it before i can even see it, and i can't disable windows defender because i'm not admin on this pc. thanks for help
